media-libs/avidemux-core/avidemux-core-2.6.20.ebuild

```diff
@@ -65,10 +65,6 @@ src_prepare() {
 		sed -i -e 's/include(admFFmpegUtil)//g' avidemux/commonCmakeApplication.cmake || die "${error}"
 		sed -i -e '/registerFFmpeg/d' avidemux/commonCmakeApplication.cmake || die "${error}"
 		sed -i -e 's/include(admFFmpegBuild)//g' avidemux_core/CMakeLists.txt || die "${error}"
-	else
-		# Avoid existing avidemux installations from making the build process fail, bug #461496.
-		sed -i -e "s:getFfmpegLibNames(\"\${sourceDir}\"):getFfmpegLibNames(\"${WORKDIR}/${P}_build/ffmpeg/source/\"):g" cmake/admFFmpegUtil.cmake \
-			|| die "Failed to avoid existing avidemux installation from making the build fail."
```

Comment:
you probably need a revbump here otherwise it'll fail to build if people do not rebuild -core